ngraph/core/include/ngraph/node_output.hpp

```diff
@@ -174,6 +178,8 @@ namespace ngraph
         bool operator<=(const Output& other) const;
         bool operator>=(const Output& other) const;
 
+        const std::unordered_set<std::string>& get_names() const;
```

Comment:
Looks like a duplicate of line 104.

Reply:
There are different specializations. Here specialization for `Output<const Node>` and in the line 104 the specialization for `Output<const Node>`

Reply:
Right, github's diff in dark mode tricked me :)